userguide/kaui/Kaui_Guide_Draft (4) (1).adoc

```diff
@@ -8,7 +8,9 @@
 :toclevels: 3
 :figure-caption!:
 :icons: font
-:imagesdir: https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/kaui
+:imagesdir: C:\_My Documents\FlowWritingLLC\Projects\Kill Bill\Documentation\killbill-docs\userguide\assets\img\kaui
+

```

Comment:
Fix `imagesdir` path to  `https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/kaui`